signal/micro/kernels/pcan.cc

```diff
@@ -0,0 +1,129 @@
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "signal/src/pcan_argc_fixed.h"
+#include "tensorflow/lite/kernels/internal/tensor_ctypes.h"
+#include "tensorflow/lite/kernels/kernel_util.h"
+#include "tensorflow/lite/micro/flatbuffer_utils.h"
+#include "tensorflow/lite/micro/kernels/kernel_util.h"
+#include "tensorflow/lite/micro/memory_helpers.h"
+#include "tensorflow/lite/micro/micro_context.h"
+
+namespace tflite {
+namespace tflm_signal {
+namespace pcan {
```

Comment:
We can drop the pcan namespace. Please also add the TODO for cleaning up the tflm_signal namespace.

Reply:
Done